packages/manager/apps/nasha-react/src/pages/details/[serviceName]/index.tsx

```diff
@@ -1,30 +1,60 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/naming-convention */
+import React, { Suspense, lazy } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useParams } from 'react-router-dom';
-import { Heading } from '@chakra-ui/react';
-import Partitions from './partitions';
-import Subscriptions from './suscriptions';
-import Informations from './informations';
-import Configurations from './configurations';
+import { OsdsText, OsdsDivider } from '@ovhcloud/ods-stencil/components/react/';
+import {
+  OdsThemeColorIntent,
+  OdsThemeTypographyLevel,
+} from '@ovhcloud/ods-theming';
+import Loading from '../../loading';
+
+const Informations = lazy(() => import('./informations'));
+const Subscriptions = lazy(() => import('./suscriptions'));
+const Configurations = lazy(() => import('./configurations'));
 
 export default function NashaReactDashboard() {
   const { t } = useTranslation('nasha-react/details/dashboard');
   const { serviceName } = useParams();
 
+  interface SectionProps {
+    title: string;
+    component: any;
+  }
+  const Section = ({ title, component: Component }: SectionProps) => (
+    <div className="col">
+      <div className="element">
+        <OsdsText
+          level={OdsThemeTypographyLevel.heading}
+          color={OdsThemeColorIntent.text}
+        >
+          {t(title)}
+        </OsdsText>
+        <div className="element">
+          <OsdsDivider separator={true} />
+        </div>
+        <Suspense fallback={<Loading />}>
+          <Component serviceName={serviceName} />
+        </Suspense>
+      </div>
+    </div>
+  );
+
   return (
-    <div>
-      <Heading as="h3" size="sm">
-        {t('title')}
-      </Heading>
-      <div>
-        <h3>{t('partitions')}</h3>
-        <Partitions serviceName={serviceName} />
-        <h3>{t('informations')}</h3>
-        <Informations serviceName={serviceName} />
-        <h3>{t('configurations')}</h3>
-        <Configurations serviceName={serviceName} />
-        <h3>{t('subscriptions')}</h3>
-        <Subscriptions serviceName={serviceName} />
+    <div className="container">
+      <div className="row">
+        <Section
+          title="nasha_dashboard_information_title"
+          component={Informations}
```

Comment:
We could put the component as children instead, it would be cleaner :

```
<Section title="nasha_***">
    <Informations serviceName={serviceName} />
</Section>
```

Or maybe it won't work as intended with Suspense ?